### ISSUE TITLE
Compilation Problem with Remix IDE Solidity

### ISSUE THREAD

**Issue by sovan17** [paco0x/amm-arbitrageur#60]
Why is it showing error during "Flashbot.sol" compilation?

**Comment by haruxe** [comment]
Have you matched the solidity compiler version in remix?

**Comment by sovan17** [comment]
Yes. It's '0.7.0' but asking to compile with 7.6 but even though its showing error for 'Import libraries and interfaces' and also with @openzeppelin/contacts.
